test/buy-tshirt.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { $, browser } from 'protractor';
+
+describe('When I will buy a t-shirt', () => {
```

Comment:
esta frase suena más a pregunta, puedes usar mejor esta `When I try to buy a t-shirt`